src/Runner.Listener/Runner.cs

```diff
@@ -451,16 +451,41 @@ private void PrintUsage(CommandSettings command)
             ext = "sh";
 #endif
             _term.WriteLine($@"
-Commands:,
- .{separator}config.{ext}          Configures the runner
- .{separator}config.{ext} remove   Unconfigures the runner
- .{separator}run.{ext}             Runs the runner interactively. Does not require any options.
+Commands:
+ .{separator}config.{ext}         Configures the runner
+ .{separator}config.{ext} remove  Unconfigures the runner
+ .{separator}run.{ext}            Runs the runner interactively. Does not require any options.
 
 Options:
+ --help     Prints the help for each command
  --version  Prints the runner version
  --commit   Prints the runner commit
- --help     Prints the help for each command
-");
+
+Config Options:
+ --unattended     Disable interactive prompts for missing arguments. Defaults will be used for missing options
+ --url string     Repository to add the runner to. Required if unattended
+ --token string   Registration token. Required if unattended
+ --name string    Name of the runner to configure (default {Environment.MachineName ?? "myrunner"})
+ --work string    Relative runner work directory (default {Constants.Path.WorkDirectory})
+ --replace        Replace any existing runner with the same name (default false)");
+#if OS_WINDOWS
+    _term.WriteLine($@" --runasservice   Run the runner as a service");
+    _term.WriteLine($@" --windowslogonaccount string   Account to run the service as. Requires runasservice");
+    _term.WriteLine($@" --windowslogonpassword string  Password for the service account. Requires runasservice");
+#endif
+    _term.WriteLine($@"
+Run Options:
```

Comment:
I would suggest taking the `--once` option out, since run once is not fully working due to race condition on the server-side.
@bryanmacfarlane for input.

Reply:
Removed it for now. It can easily be added back when we are ready to release/support it

Reply:
If we add it back, it would likely be something like --ephemeral.  keep it gone for now.  we should also delete the code supporting once in a separate PR since it was the wrong pattern